game/Dockerfile

```diff
@@ -1,12 +1,7 @@
-FROM l2py_data_models
+FROM l2py-common as l2py-game
```

Comment:
Please don't use dashes in container names.
Plus if you don't explicitly need something from parent image its not necessary to use ```as l2py-game```